cdap-security/src/main/java/co/cask/cdap/security/guice/SecureStoreModules.java

```diff
@@ -50,70 +53,65 @@
  *                    - < Hadoop 2.6        - Invalid, throws IllegalArgumentException. (mode not supported)
  *      - Standalone  - > Hadoop 2.6        - Invalid, throws IllegalArgumentException. (mode not supported)
  *                    - < Hadoop 2.6        - Invalid, throws IllegalArgumentException. (mode not supported)
- *      - Distributed - > Hadoop 2.6        - Valid, returns KMSSecureStore
+ *      - Distributed - > Hadoop 2.6        - Valid, returns KMSSecureStoreService
```

Comment:
Can you open a JIRA to have the cleanup? The final goal is to turn the KMS and File implementation to `SecretManager` extension based.